.gitignore

```diff
@@ -78,3 +78,6 @@ temp/
 
 dist/
 build/
+
+# Jetbrains IDEs
+.idea/
```

Comment:
Can you please remove this? I'd prefer we not introduce changes orthogonal to the primary fix in the same PR.

Reply:
Thank you. I reverted that changes.